web-client/iron-remote-desktop/src/services/remote-desktop.service.ts

```diff
@@ -36,6 +36,7 @@ export class RemoteDesktopService {
     private cursorHasOverride: boolean = false;
     private lastCursorStyle: string = 'default';
     private enableClipboard: boolean = true;
+    private _autoClipboard: boolean = true;
```

Comment:
question: Why the `_` prefix? Does not seem to be the style we used so far (I also believe the `private` keyword is enough)